gem/README.md

```diff
@@ -0,0 +1,68 @@
+# EarlGrey
+
+[![Gem Version](https://badge.fury.io/rb/earlgrey.svg)](https://rubygems.org/gems/earlgrey)
+
+Automatically installs EarlGrey. Supports carthage, cocoapods, and Swift.
+
+```
+$ earlgrey help install
+Usage:
+  earlgrey install -t, --target=TARGET
+
+Options:
+  -p, [--project=PROJECT]            # Project
+  -t, --target=TARGET                # EarlGrey
+  -s, [--scheme=SCHEME]              # EarlGrey.xcscheme
+      [--swift], [--no-swift]
+                                     # Default: true
+      [--carthage], [--no-carthage]
+                                     # Default: true
+
+Installs EarlGrey into an Xcode unit test target
+```
+
+## Usage
+
+Create new Target `iOS Unit Testing Bundle` then add a new scheme
```

Comment:
<img width="422" alt="hmm_ui_vs_unit" src="https://cloud.githubusercontent.com/assets/1173057/15074146/6e38e5c6-136b-11e6-8298-984d1d9c4287.png">

The existing docs refer to the [unit target](https://github.com/google/EarlGrey/blob/master/docs/install-and-run.md#step-1-set-up-a-test-target) because EarlGrey does not work in a UI target. Has this changed?